Sk add js form getter

shelbykauth opened the pull request.

shelbykauth committed:
Merge remote-tracking branch 'refs/remotes/mariekauth/master'

shelbykauth committed:
add form getter

mariekauth commented on scripts/model-writer.js:
```diff
@@ -17,6 +17,29 @@ $(document).ready(function(){
+    var args = $(".form-group.args");
+    var names = $("[name='arg_name[]'");
+    var types = $("[name='arg_type[]'");
+    console.log(args);
```
Please remove the console.log(args), once you were done testing, it was no longr needed. Also leave this as a blank line.

mariekauth commented on scripts/model-writer.js:
```diff
@@ -17,6 +17,29 @@ $(document).ready(function(){
+        newArg.type = types[i].value;
+        returnObject.args.push(newArg);
+    }
+    return returnObject;
```
Please leave ablank line before your return.

mariekauth reviewed (changes requested):
Please update the following.
Then:
git status
git add {file}
git commit -m 'Code Review'
git rebase -i upstream/master

In vi or vim, mark the additional commit with -f (for fix-up).

git push origin sk_branchname -f
(This force pushes the branch because it was rebased)

shelbykauth committed:
Removed console.log, changed spacing

Added blank line before return line.
Added blank line before for loop variables
